clients/fides-js/src/docs/fides-options.ts

```diff
@@ -333,4 +334,13 @@ export interface FidesOptions {
   fides_unsupported_repeated_script_loading?:
     | "enabled_acknowledge_not_supported"
     | "disabled";
+
+  /**
+   * Sets a suffix for the `fides_consent` cookie that Fides uses to store consent preferences.
+   *
+   * If provided the cookie name will be `fides_consent_given_suffix_value`.
+   *
```

Comment:
Consider clarifying it’s the actual suffix value using an example here:

```suggestion
   * If provided the cookie name will be `fides_consent_given_suffix_value`.
   * (e.g., fides_consent_mycompany if suffix is "mycompany").
   *
```